libs/agora/ui/src/lib/footer/footer.component.html

```diff
@@ -1,39 +1,23 @@
-<footer class="mat-typography">
-  <div class="row">
-    <div class="col-8 about-oc">
-      <img class="logo" src="/agora-assets/images/agora-color.svg" alt="Agora logo" />
-      <ul class="footer-link-group">
-        <li><a routerLink="/about">About</a></li>
-        <li><a routerLink="/team">Meet Our Team</a></li>
-        <li>
-          <a
-            [href]="apiDocsUrl"
-            title="Interactive documentation for OC API"
-            rel="noopener noreferrer"
-            target="_blank"
-            >API Docs</a
-          >
-        </li>
-        <li>
-          <a
-            href="https://github.com/Sage-Bionetworks/sage-monorepo/issues/new/choose"
-            title="Share feedback about the OC app"
-            rel="noopener noreferrer"
-            target="_blank"
-            >Submit a Feature/Bug</a
-          >
-        </li>
-      </ul>
-    </div>
-    <div class="col-4 app-info">
-      <ul>
-        <li>App version: {{ appVersion }}</li>
-      </ul>
-    </div>
+<footer>
+  <div class="footer-nav" *ngIf="navItems.length">
```

Comment:
How about starting to use the new control flow?

```html
@if (navItems.length > 0) {
  <div class="footer-nav">
    <!-- Your content here -->
  </div>
}
```

Reply:
Good idea.  Now is a good time as any!